core/src/cluster_info.rs

```diff
@@ -94,13 +100,21 @@ pub enum ClusterInfoError {
     BadGossipAddress,
 }
 #[derive(Clone)]
+pub struct DataBudget {
+    bytes: usize,        // amount of bytes we have in the budget to send
+    last_timestamp: u64, // Last time that we upped the bytes count,
```

Comment:
```suggestion
    last_timestamp_ms: u64, // Last time that we upped the bytes count,
```
or use an `Instant`?